libs/sdk-ui-kit/src/Header/UpsellButton.tsx

```diff
@@ -0,0 +1,21 @@
+// (C) 2021 GoodData Corporation
+import React from "react";
+
+/**
+ * @internal
+ */
+export interface IUpsellButtonProps {
+    onUpsellButtonClick?: (e: React.MouseEvent) => void;
+}
+
+/**
+ * @internal
```

Comment:
I think we don't need `@internal` if we don't want the component to be public, similarly to other components like [HeaderMenu](https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-ui-kit/src/Header/HeaderMenu.tsx), [HeaderHelp](https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-ui-kit/src/Header/HeaderHelp.tsx) or [HeaderAccount](https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-ui-kit/src/Header/HeaderAccount.tsx)

Reply:
I updated it thanks